hassio/api/proxy.py

```diff
@@ -117,7 +113,7 @@ def _check_access(self, request):
 
         try:
             client = await self.sys_websession_ssl.ws_connect(
-                url, heartbeat=60, verify_ssl=False)
+                url, heartbeat=30, verify_ssl=False)
```

Comment:
We still need this?

Reply:
I think it make no pains and it could help to detect faster wrong connections